<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="PointsView" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>Processing</string>
    </property>
    <addaction name="actionCVT"/>
    <addaction name="actionCVT_with_line"/>
    <addaction name="separator"/>
    <addaction name="actionRecalculate_z_coordinate"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionNumber_of_iterations"/>
    <addaction name="actionReset_move_counter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProcessing"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCVT">
   <property name="text">
    <string>CVT</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRecalculate_z_coordinate">
   <property name="text">
    <string>Recalculate z coordinate</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCVT_with_line">
   <property name="text">
    <string>CVT with line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionNumber_of_iterations">
   <property name="text">
    <string>Number of iterations</string>
   </property>
  </action>
  <action name="actionReset_move_counter">
   <property name="text">
    <string>Reset move counter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PointsView</class>
   <extends>QWidget</extends>
   <header>gui/pointsview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
